timely_beliefs/beliefs/classes.py

```diff
@@ -1467,6 +1492,124 @@ def resample_events(
 
         return df
 
+    @hybrid_method
+    def form_beliefs(
```

Comment:
Why not "forecast_beliefs"? 
We pass in a "forecaster", so that's clearly what is happening, and we can be more specific.

Reply:
This will get pretty meta, but a forecast **is** a belief, and a belief is not something you typically forecast, but rather is something you form.

Unless I forecast that you will believe I am mistaken, in which case my belief is the forecast and your belief is the event my belief is about.
```
                                                                                   event_value  sensor
event_start               belief_time               source cumulative_probability             
2022-09-16 13:20:00+02:00 2022-09-16 15:10:00+02:00 Felix  0.5                           False Nicolas
2022-09-16 13:25:00+02:00 2022-09-16 15:10:00+02:00 Felix  0.5                            True Nicolas
```

:)